proto/crconsent/request.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+package metabelarus.mbcorecr.crconsent;
+
+option go_package = "github.com/metabelarus/mbcorecr/x/crconsent/types";
+
+import "gogoproto/gogo.proto";
+
+enum RequestType {
+  INVITE_PACK = 0; 
+  SERVICE_PERMISSIONS = 1;
```

Comment:
For me it's not clear, how we store the list of possible permissions itself. Actually the permissions will be application specific hardcoded thing. So some separate enum is a good idea for them.